pmd-apex/src/test/resources/net/sourceforge/pmd/lang/apex/rule/codestyle/xml/MethodNamingConventions.xml

```diff
@@ -34,18 +34,7 @@ public class Foo {
 }
 		]]></code>
     </test-code>
-    
-    <test-code>
-        <description>#1343 MethodNamingConventions for overrided methods</description>
-        <expected-problems>0</expected-problems>
-        <code><![CDATA[
-public class MethodNamingConventions implements SomeInterface {
```

Comment:
Any particular reason, why you removed this test case?

It seems, in Apex, there is no Override annotation, but a override keyword:

```
public class MethodNamingConventions implements SomeInterface {
    public override void _foo() { }
}
```

Reply:
yes, I can confirm, it compiles then....